### PR TITLE
feat: add grpc api related to the block

### PR DESCRIPTION
**Pull request by brew0722**
## Description

- Add below API to integrate with grpc.
`BlockByHash`, `BlockResultsByHeight`

- Add prove parameter to `GetTxsEvent` API.
In the existing case, prove is hard-coded to `true`, which shows slow performance.

- Migrate swagger-gen.

---

Before we can merge this PR, please make sure that all the following items have been
checked off. If any of the checklist items are not applicable, please leave them but
write a little note why.

- [ ] Targeted PR against correct branch (see [CONTRIBUTING.md](https://github.com/cosmos/cosmos-sdk/blob/master/CONTRIBUTING.md#pr-targeting))
- [ ] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.
- [ ] Code follows the [module structure standards](https://github.com/cosmos/cosmos-sdk/blob/master/docs/building-modules/structure.md).
- [x] Wrote unit and integration [tests](https://github.com/cosmos/cosmos-sdk/blob/master/CONTRIBUTING.md#testing)
- [ ] Updated relevant documentation (`docs/`) or specification (`x/<module>/spec/`)
- [x] Added relevant `godoc` [comments](https://blog.golang.org/godoc-documenting-go-code).
- [ ] Added a relevant changelog entry to the `Unreleased` section in `CHANGELOG.md`
- [ ] Re-reviewed `Files changed` in the Github PR explorer
- [ ] Review `Codecov Report` in the comment section below once CI passes

### CODE REVIEW

**Review by egonspace — APPROVED** [review]
LGTM

**Comment by egonspace** [comment]
`CHANGELOG` also needs to be updated.

**Comment by zemyblue** [comment]
@slave5vw Could you change the location of wasm's protobuf files?

**Comment by brew0722** [comment]
@zemyblue 
The legacy wasm proto paths will be resolved in cosmwasm/wasmd 0.17.0 migration PR.
https://github.com/CosmWasm/wasmd/tree/v0.17.0/proto/cosmwasm/wasm/v1beta1
https://github.com/line/lfb-sdk/pull/249

> @slave5vw Could you change the location of wasm's protobuf files?

File: client/docs/config.json [hunk 1]
```diff
@@ -13,79 +13,79 @@
       }
     },
     {
-      "url": "./tmp-swagger-gen/cosmos/auth/v1beta1/query.swagger.json",
```
Comment (zemyblue) [inline client/docs/config.json:16]: How about change the title of swagger info above?
`Cosmos SDK - Legacy ~~~` -> ` LFB SDK - Legacy`

Comment (brew0722) [reply client/docs/config.json:16]: thks! changed it

**Review by zemyblue — APPROVED** [review]

**Pull request merged**